runtime/oti/j9nonbuilder.h

```diff
@@ -5363,6 +5365,7 @@ typedef struct J9JavaVM {
 #ifdef J9VM_OPT_VALHALLA_VALUE_TYPES
 	UDATA valueFlatteningThreshold;
 #endif /* defined(J9VM_OPT_VALHALLA_VALUE_TYPES) */
+	U_64 extendedRuntimeFlags2;
```

Comment:
please put this with the other runtimeFlags field near the top of the structure.  We've seen surprising overheads reading this far into the J9JavaVM for status flags

Reply:
Also not a fan of U_64. Until we get smart and start using proper bitfields, let's keep flags fields as U_32 or UDATA.

Reply:
Not UDATA as it wastes 1/2 the range on 64 bit.